### DISCUSSION TITLE
[Mono-Symbolicate] .Net Maui iOS crash

### DISCUSSION THREAD

**Discussion by PetrePopescu** [dotnet/maui#10260] [Q&A]
I am trying to use the mono-symbolicate tool to work with a **Maui iOS app** but the **bin** folder does not seem to contain any **msym** folder.

Is there any config/workaround to get the **msym** to show up?

[SymbolTestApp.zip](https://github.com/dotnet/maui/files/9624897/SymbolTestApp.zip)

**Comment by mkshiblu** [comment]
I ran into the same issue. Cannot generate C# stack-trace locally.
